src/sentry/templatetags/sentry_assets.py

```diff
@@ -0,0 +1,39 @@
+from django.template import Library
```

Comment:
from __future__ import absolute_import

(how did you manage to avoid the linter)